tests/chainerx_tests/conftest.py

```diff
@@ -123,14 +123,17 @@ def is_module(request):
     (2, 0, 3),
 ]
 
-_shapes_as_tuple_or_int = _shapes + [0, 1, 5]
+_shapes_as_sequence_or_int = (
+    _shapes
+    + list(map(lambda t: list(t), _shapes))
```

Comment:
I don't think we need `list` conterparts for all the shapes.
How about adding a single or two parameters?